plugins/kaggle/api_token.go

```diff
@@ -0,0 +1,81 @@
+package kaggle
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://www.kaggle.com/docs/api"),
+		ManagementURL: sdk.URL("https://www.kaggle.com/settings/account"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "API Token used to authenticate to Kaggle.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 32,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+			{
+				Name:                fieldname.Username,
+				MarkdownDescription: "Username to authenticate to Kaggle.",
+				Secret:              true,
```

Comment:
The username isn't the authentication secret, and we usually do not conceal these by default.

Fields that aren't marked secret are still end-to-end encrypted in 1Password and only available to folks you've shared the item with. The difference is in how we handle them in the client, like showing the plaintext by default on screen.

```suggestion
				Secret:              false,
```